### ISSUE TITLE
How can i use multi bus system

### ISSUE THREAD

**Issue by godveryday** [renode/renode#718]
I want to configurate HW with multibus system
ex)
sysbus1
sysbus2
sysbus3

Is it possible?

**Comment by PiotrZierhoffer** [comment]
No, at the moment there is only one bus. You can define which parts of the system have access where. For example, see https://github.com/renode/renode/blob/master/tests/unit-tests/per-core-registration.repl where some memories are marked as accessible by only one core
